sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_constants.py

```diff
@@ -33,3 +33,5 @@ class SchemaFormat(str, Enum):
 
     AVRO = "avro"
     """Represents the Apache Avro schema format."""
+
+SUPPORTED_API_VERSIONS = ["2021-10"]
```

Comment:
check here for api version:

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/search/azure-search-documents/azure/search/documents/_api_versions.py